python/private/patch_whl.bzl

```diff
@@ -97,4 +101,19 @@ def patch_whl(rctx, *, python_interpreter, whl_path, patches, **kwargs):
             ),
         )
 
+    if record_patch.exists:
```

Comment:
Record patching is effectively required, right, because it has the hashes? I'm just thinking warning about this might be spammy.

Reply:
resolve in chat. I didn't realize .exists is a function call to stat the path, not a simple attribute lookup from the time path() was called.